src/views/AboutPage/components/common/SectionDescription/index.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled';
+import { ReactNode } from 'react';
+
+const SectionDescription = ({ children }: { children: ReactNode }) => {
```

Comment:
React 18 types 에 `PropsWithChildren<T>` 타입이 추가되었더군요 !!

```suggestion
const SectionDescription = ({ children }: PropsWithChildren) => {
```

Reply:
요것이 .... 다른 프롭이 없으면 빈 타입을 T 자리에 주어야 하는데 그것의 방법을 못 찾아서 직접 선언하였습니다 🥲🥲